.github/workflows/ci.yml

```diff
@@ -309,7 +308,7 @@ jobs:
 
   docs:
     needs: build
-    if: github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch'
+    if: contains(fromJson('["refs/heads/main"]'), github.ref) || github.event_name == 'workflow_dispatch'
```

Comment:
Since the list of branches has only one element I think we can replace this with:

```
if: github.ref == 'refs/heads/main'
```

Reply:
Oh true, the example was including a ref to master. Thanks.